kotsadm/web/src/components/SecureAdminConsole.jsx

```diff
@@ -147,6 +148,34 @@ class SecureAdminConsole extends React.Component {
     this.loginText.current.style.fontSize = newFontSize;
   }
 
+  getLoginInfo = async () => {
+    try {
+      const response = await fetch(`${window.env.API_ENDPOINT}/login/info`, {
+        headers: {
+          "Content-Type": "application/json",
+        },
+        method: "GET",
+      });
+
+      if (!response.ok) {
+        const res = await response.json();
+        if (res.error) {
+          throw new Error(`Unexpected status code ${response.status}: ${res.error}`);
+        }
+        throw new Error(`Unexpected status code ${response.status}`);
+      }
+
+      const loginInfo = await response.json();
+      this.setState({ loginInfo });
+
+      return loginInfo;
+    } catch(err) {
+      console.log(err);
```

Comment:
I'll prefer to have here some error messaging. like we have in some other fetch requests. 
instead of throwing new errors here: https://github.com/replicatedhq/kots/pull/1299/files#diff-fc7da0c2d73558c49ab76272822c89de39e9c28469dafce793f20aa1b4f333e0R163-R165
save those messages in state and show it somewhere in UI.